utils/releaser/src/windows/desktop.rs

```diff
@@ -29,7 +29,15 @@ fn build_x86() -> CliResult<()> {
 
     Command::new("cargo")
         .env("LB_TARGET", "x86_64-pc-windows-msvc")
-        .args(["build", "-p", "winstaller", "--release", "--target=x86_64-pc-windows-msvc"])
+        .args([
+            "build",
+            "-p",
+            "winstaller",
+            "--release",
+            "--target=x86_64-pc-windows-msvc",
+            "--feature",
+            "releaser",
+        ])
```

Comment:
I hope this works. Not sure how this should be tested

Reply:
Yeah the easiest way would be to just run the releaser -- you can `cargo install --path .` in the `releaser` folder and verify that it _fails successfully_ lol. You can try it on `master`, it should compile a few things and then fail to upload to github (because you're not authorized, or because 0.9.0 exists already or other such error). 

Then you can: 
1. switch to your branch 
2. execute `cargo clean` 
3. comment out `--feature releaser` 
4. verify you get a different error message (probably something like the original error you were getting -- `LB_TARGET` not found)
5. bring back `--feature releaser`, run and get the github error again (the feature flag worked)

Reply:
I think you telling me the outcome of ^ is all that stands in the way of this being merged

Reply:
I'm not sure of a great way to test it. After setting GITHUB_TOKEN to a dummy value, the command that requires the feature flag does get run before failing at the github portion. 

However, it passes with and without the feature flag. One different outcome I noticed though is that winstaller file is much larger after running the releaser when compiled with the feature flag, compare to when it is compiled without the feature flag. 

Here are screenshots to show that
Result of releaser when compiled with the flag

Result of releaser when compiled without the flag
![image](https://github.com/lockbook/lockbook/assets/4559311/65a477ce-ca77-4c27-aac9-03d6fde1f583)

Reply:
a successful `winstaller` build contains the full `egui` client within it a la [`include_bytes!`](https://doc.rust-lang.org/std/macro.include_bytes.html). Like I mentioned [here](https://discord.com/channels/1014184997751619664/1222701324462854187/1227625533743824976) ideally the `277kb` outcome is an impossibility. But I know we're already juggling a lot of constraints at this point, and just getting a windows person to be able to clippy locally is more important.